backend/gameplay/consumers.py

```
@@ -446,10 +446,10 @@ async def update_position(self, chess_game_model: ChessGame, move_info: dict):
 		chess_game_model.parsed_board_placement = new_board_placement
 		chess_game_model.current_player_turn = new_side_to_move
 
-		await self.append_to_position_list(chess_game_model, move_info, move_type)
+		await self.update_captured_material(original_board_placement, move_info, original_en_passant_target_square, chess_game_model)
```

Pylint (3.3.7)

[convention] Line too long (127/100)

(C0301)